js/index.js

```diff
@@ -0,0 +1,37 @@
+
+const replaceSVGTemplatePlaceholders = (template, data) => {
```

Comment:
can we expect vcdata as input and add downloading the template from url inside the library , so that all implementers dont have to download it and send the vcdata, which internally has svg string
Since it is a spec, we can expect it to be present which call reaches library.

Reply:
Update the library to fetch SVG URL as well.